File: sct_source/SourceCode/FE_SoCongThuong/Angular/src/app/modules/app-pages/management-electricity-activities-page/_models/management-electricity-activities.model.ts
import { BaseModel } from '../../../../_metronic/shared/crud-table/models/base.model';

export interface ManagementElectricityActivitiesModel extends BaseModel {
  managementElectricityActivitiesId: string;
  projectName: string;
  districtId: string;
  wattage: number;
  maxWattage: number;
  type: number | null;
  dateOfAcceptance: string;
  connectorAgreement: string;
  powerPurchaseAgreement: string;
  anotherContent:string;
}

File: sct_source/SourceCode/API_SoCongThuong/Models/ManagementElectricityActivitiesModel.cs
namespace API_SoCongThuong.Models
{
    public class ManagementElectricityActivitiesModel
    {
        public Guid ManagementElectricityActivitiesId { get; set; }
        public string ProjectName { get; set; }
        public Guid DistrictId { get; set; }
        public string? DistrictName { get; set; }
        public double Wattage { get; set; }
        public double MaxWattage { get; set; }
        public int Type { get; set; }
        public string? TypeName { get; set; }
        public DateTime DateOfAcceptance { get; set; }
        public string ConnectorAgreement { get; set; }
        public string PowerPurchaseAgreement { get; set; }
        public string AnotherContent { get; set; }
    }
}